WordPress/src/androidTest/java/org/wordpress/android/e2e/flows/LoginFlow.java

```
@@ -38,6 +38,7 @@ public LoginFlow chooseContinueWithWpCom() {
     public LoginFlow enterEmailAddress(String emailAddress) {
         // Email Address Screen – Fill it in and click "Continue"
         // See LoginEmailFragment
+        clickOn(R.id.input);
         populateTextField(R.id.input, emailAddress);
         clickOn(R.id.login_continue_button);
```

Got a question based on the PR description. Does it mean that the `Continue` and `Continue with Google` buttons use the same identifier? Probably out of the scope of this PR, but I think it should fail with a not found/button disabled error instead of clicking another button

---

No, they are using different identifiers.

But somehow, the `Google Ligin` pop-up in question appears on the screen. My quite unfounded guess is that the test runner (and maybe some additional logic coming from FTL 🤔) still tries to do things, after it fails to tap the normal `Continue` button (which is likely disabled). This change  above only reduced the probability of usual `Continue` to be disabled, by tapping into email edit field before entering the text.